crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -47,9 +51,11 @@ impl ProtocolFee {
         {
             if let Some(partner_fee) = validated_app_data.protocol.partner_fee {
                 let fee_policy = vec![Policy::Volume {
-                    factor: FeeFactor::partner_fee_capped_from(
+                    factor: FeeFactor::try_from_capped(
                         partner_fee.bps.into_f64() / 10_000.0,
-                    ),
+                        self.max_partner_fee.into(),
+                    )
+                    .unwrap(),
```

Comment:
nit: I think it would be nice if we already panicked on startup in case `fee_policy_max_partner_fee` is >= 1. 
Otherwise, in case of bad configuration, we may have to wait until a user places an order with a partner fee (with some bad luck during the night on a weekend) before we run into a crash loop. It would be better to crashloop during the rollout.

Reply:
This should already be the case since the CLI arg gets parsed into a FeeFactor that enforces that invariant in its FromStr implementation.

Reply:
@fleupold, as @MartinquaXD said, it is already imposed by the `FeeFactor` and it panics on startup in case of bad configuration.